apps/foundation-demo/src/app/core/loading-icon/loading-icon/loading-icon.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: '[loading-icon]',
```

Comment:
Can we also do `<loading-icon>` as opposed to `<div loading-icon></div>` I don't see a reason for a HostComponent vs. custom component?

Reply:
done